README.md

```
@@ -0,0 +1,184 @@
+## Team
+
+<div align="center">
+<table>
+    <tr>
+        <th> 👑 <a href="https://github.com/Jeong-Ag">권동희</a></th>
+        <th> <a href="https://github.com/ljh0608">장민수</a></th>
+        <th> <a href="https://github.com/kojesung">양승혜</a></th>
+        <th> <a href="https://github.com/heeyyeon">송민하</a></th>
+    </tr>
+    <tr>
+    	<td><img width="140" height="140" alt="동희" src="https://github.com/user-attachments/assets/5ac3c9b6-3994-4b3a-a1a7-b486e484f1ed" /></td>
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 5

(MD010, no-hard-tabs)